nano/templates/atm.tmpl

```diff
@@ -0,0 +1,151 @@
+<!-- 
+Title: ATM
+Used In File(s): /code/modules/economy/ATM.dm
+-->
+
+<!--
+#define NO_SCREEN					0
+#define CHANGE_SECURITY_LEVEL		1
+#define TRANSFER_FUNDS				2
+#define VIEW_TRANSACTION_LOGS		3
+-->
+
+<head>
```

Comment:
This is improper HTML, templates are inside the body tag, a head tag should not be defined. Just drop the <head> and leave <style>.